app/src/common/shared/com/igalia/wolvic/VRBrowserActivity.java

```diff
@@ -268,7 +268,8 @@ private boolean callOnAudioManager(Consumer<AudioManager> fn) {
         @Override
         public void onGlobalFocusChanged(View oldFocus, View newFocus) {
             Log.d(LOGTAG, "======> OnGlobalFocusChangeListener: old(" + oldFocus + ") new(" + newFocus + ")");
-            triggerHapticFeedback();
+            // TODO: Which controller should we send the haptic feedback to ?
+            triggerHapticFeedback(0);
```

Comment:
I think we might not need this at all?

In which cases not covered by the usages in KeyboardWidget is this triggered?

Reply:
Not sure, I didn't want to remove the call as part of this change.